root/user/components/UserHasNotUsedTag.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import EditorLink from '../../static/scripts/common/components/EditorLink';
+import TagLink from '../../static/scripts/common/components/TagLink';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+import expand2text from '../../static/scripts/common/i18n/expand2text';
+import {headingsText} from '../../tag/EntityList';
+
+type Props = {
+  +entityType?: string,
+  +showDownvoted: boolean,
+  +tag: TagT,
+  +user: EditorT,
+};
+
+const noDownvotesText = {
+  area: N_l('{user} has not voted against any “{tag}” tags for areas.'),
+  artist: N_l('{user} has not voted against any “{tag}” tags for artists.'),
+  event: N_l('{user} has not voted against any “{tag}” tags for events.'),
+  instrument: N_l(
+    '{user} has not voted against any “{tag}” tags for instruments.',
+  ),
+  label: N_l('{user} has not voted against any “{tag}” tags for labels.'),
+  place: N_l('{user} has not voted against any “{tag}” tags for places.'),
+  recording: N_l(
+    '{user} has not voted against any “{tag}” tags for recordings.',
+  ),
+  release: N_l('{user} has not voted against any “{tag}” tags for releases.'),
+  release_group: N_l(
+    '{user} has not voted against any “{tag}” tags for release groups.',
+  ),
+  series: N_l('{user} has not voted against any “{tag}” tags for series.'),
+  work: N_l('{user} has not voted against any “{tag}” tags for works.'),
+};
+
+const noTagsText = {
+  area: N_l('{user} has not tagged any areas with “{tag}”.'),
+  artist: N_l('{user} has not tagged any artists with “{tag}”.'),
+  event: N_l('{user} has not tagged any events with “{tag}”.'),
+  instrument: N_l('{user} has not tagged any instruments with “{tag}”.'),
+  label: N_l('{user} has not tagged any labels with “{tag}”.'),
+  place: N_l('{user} has not tagged any places with “{tag}”.'),
+  recording: N_l('{user} has not tagged any recordings with “{tag}”.'),
+  release: N_l('{user} has not tagged any releases with “{tag}”.'),
+  release_group: N_l(
+    '{user} has not tagged any release groups with “{tag}”.',
+  ),
+  series: N_l('{user} has not tagged any series with “{tag}”.'),
+  work: N_l('{user} has not tagged any works with “{tag}”.'),
+};
```

Comment:
Please provide a URL example using these strings, if any.

Reply:
http://localhost:5000/user/reosarevok/tag/classical/release /

Reply:
Thanks, how do you reach these pages?

Reply:
I edited the comment in the meantime to specify that, sorry! :)

Reply:
Found it: You click on “See all {num} works” under the short list of 10 first works available from https://musicbrainz.org/user/jesus2099/tag/instrumental

So there seems to be two issues:
* Navigation: There should be a way to go back to `/tag/instrumental` from `/tag/instrumental/work`.
* Description: Why are these strings entity type-specific and neither the header `Vote for/against “{tag}”` nor its following paragraph? If one of these strings should be entity type-specific that should probably be the header, other strings can probably be entity type-neutral, like what we have for subscriptions already.

Reply:
Both should be, because just saying "jesus2099 has not voted against any “instrumental” tags" would be a lie :) But that's fair, other lines here should also be specific, at least the "Showing entities where" bit.